sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapi/AADJwtBearerTokenAuthenticationConverterTest.java

```diff
@@ -56,18 +59,21 @@ public void testExtractDefaultScopeAuthorities() {
             .getPrincipal();
         assertThat(principal.getAttributes()).isNotEmpty();
         assertThat(principal.getAttributes()).hasSize(2);
+        assertThat(principal.getAuthorities()).hasSize(2);
```

Comment:
this test method name should be changed too since the default authorities are not only scopes anymore.

Reply:
Aren't we supposed to get 4 authorities here?

Reply:
Oh, I see..